YAXLibTests/SerializationTest.cs

```diff
@@ -171,8 +180,13 @@ public void CultureChangeTest()
   <Dec1>192389183919123.18232131</Dec1>
   <Date1>10/11/2010 18:20:30</Date1>
 </CultureSample>";
+            Assert.That(expected, Is.EqualTo(usSerResult), "Checking US is as expected!");
 
-            Assert.That(expected, Is.EqualTo(usResult), "Checking US is as expected!");
+            // deserialization results
+            Assert.AreEqual(0, faDesResult?.CompareTo(frDesResult), "Comparing deserialized FA and FR");

```

Comment:
Ah then I would be better to generate equals(). Resharper can do that for you :)